From 48d62a7c6f94a9f9537facbd2465db30ce3a57d7 Mon Sep 17 00:00:00 2001
From: Volker Lendecke <vl@samba.org>
Date: Mon, 4 Apr 2016 13:43:02 +0200
Subject: [PATCH 1015/1096] nwrap: Fix the build on Solaris

BUG: https://bugzilla.samba.org/show_bug.cgi?id=11816

Signed-off-by: Volker Lendecke <vl@samba.org>
Reviewed-by: Andreas Schneider <asn@samba.org>

Autobuild-User(master): Volker Lendecke <vl@samba.org>
Autobuild-Date(master): Tue Apr  5 08:57:06 CEST 2016 on sn-devel-144

(cherry picked from commit ff6b49beeb5df30f4e243a97d2e6218ec497e9ad)
(cherry picked from commit 969ddf12550e3c995a9bc2c016638874d8cef50b)
---
 lib/nss_wrapper/wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/nss_wrapper/wscript b/lib/nss_wrapper/wscript
index 6c3d7f7..c727980 100644
--- a/lib/nss_wrapper/wscript
+++ b/lib/nss_wrapper/wscript
@@ -62,7 +62,7 @@ def configure(conf):
                                define='HAVE_SOLARIS_GETPWUID_R', headers='unistd.h pwd.h')
         conf.CHECK_C_PROTOTYPE('getgrent_r',
                                'struct group *getgrent_r(struct group *src, char *buf, int buflen)',
-                               define='SOLARIS_GETGRENT_R', headers='unistd.h grp.h')
+                               define='HAVE_SOLARIS_GETGRENT_R', headers='unistd.h grp.h')
         conf.CHECK_C_PROTOTYPE('getgrnam_r',
                                'int getgrnam_r(const char *name, struct group *grp, char *buf, int buflen, struct group **pgrp)',
                                define='HAVE_SOLARIS_GETGRNAM_R', headers='unistd.h grp.h')
-- 
2.9.0

